ci-operator/step-registry/ipi/conf/cpu-partitioning/ipi-conf-cpu-partitioning-ref.yaml

```diff
@@ -0,0 +1,13 @@
+ref:
+  as: ipi-conf-cpu-partitioning
+  from_image:
+   namespace: ocp
+   name: "4.12"
```

Comment:
Is 4.12 correct here?

Reply:
I had assumed that was fine since it's just doing a simple yq transform, but that's a good call out, let me keep it up to date with the min version this feature touches.